site/en/tutorials/keras/regression.ipynb

```diff
@@ -626,7 +626,7 @@
         "id": "Z3q1I9TwnRSC"
       },
       "source": [
-        "Having configured the training configured, use Keras `Model.fit` to execute the training for 100 epochs:"
+        "Having configured the training configuration, use Keras `Model.fit` to execute the training for 100 epochs:"
```

Comment:
Since the previous Markdown block says: "Once the model is built, configure the training procedure using the Keras `Model.compile` method...", maybe we could shorten this:

```suggestion
        "Use Keras `Model.fit` to execute the training for 100 epochs:"
```